app/common/elasticsearchclient.py

```diff
@@ -2851,21 +2796,232 @@ def _get_complex_datasource_filter(self, datasources, bol):
             }
         return dict()
 
+
+class AggregationUnitAbstract(AggregationUnit):
+    def build_query_filter(self):
+        if self.filter is not None:
+            self.query_filter = self._get_complex_abstract_kw_filter(self.filter,
+                                                                     BooleanFilterOperator.OR)
+
+    def build_agg(self, filters):
+        self.agg = self._get_abstract_keywords_facet_aggregation(filters)
+
+    def _get_abstract_keywords_facet_aggregation(self, filters):
+        return {
+            "significant_terms": {"field": "private.facets.abstract_lemmas.value"},
```

Comment:
Yes, that is true in the context of the web app, making calls to this endpoint only for a given target-disease pair. But from the API point of you we should allow querying for a term independently of a specific target-disease pair, thus the cardinality sub aggs are needed in the facet.